Resources/Locale/en-US/round-end/round-end-system.ftl

```diff
@@ -1,6 +1,7 @@
 ## RoundEndSystem
 
 round-end-system-shuttle-called-announcement = An emergency shuttle has been sent. ETA: {$time} {$units}.
+round-end-system-shuttle-auto-called-announcement = A departure shuttle has been sent. ETA: {$time} {$units}. Recall the shuttle to extend the shift.
```

Comment:
```suggestion
round-end-system-shuttle-auto-called-announcement = An automatic crew shift change shuttle has been sent. ETA: {$time} {$units}. Recall the shuttle to extend the shift.
```

the auto shuttle has always (to my knowledge) been referred to as a "crew shift change" shuttle, as that is what it sorta represents. I also added automatic since it happens without any input by a player right now